docs/data/install-xroot-client.md

```diff
@@ -1,97 +1,196 @@
+This page provides instructions for installing and using the XRootD client software to access data using XRootD.
 
-This page provides instructions for using a XRootD client to connect to a XRootD storage system.
+As a user you have three different ways to interact with XRootD: 
 
-Installing a XRootD Client
---------------------------
+1. Using the XRootD client tools command 
+1. Using LD\_PRELOAD to use XRootD libraries with Unix tools
+1. Using a XRootDFS FUSE mount to mount a local XRootD data store
 
-As a user you have three different ways to interact with XRootd: a file system mounted with XRootDFS, the application `xrdcp`, LD\_PRELOAD.
+We'll show how to install the XRootD client software and use all three mechanisms to access data.
 
-To use a XRootD file system mounted with XRootDFS you don't need any special setup, you will need the XRootD FUSE module.
+!!! Note
+    Only the client tools method should be used to access XRootD systems across a WAN link.
 
-To use `xrdcp` you need to have it in your path (e.g. `source $INSTALL_DIR/setup.sh`, where `$INSTALL_DIR` is the XRootD installation directory)
 
-The use of LD\_PRELOAD is not recommended if you have any alternative available.
+Before Starting
+---------------
 
-### Install Repositories 
+Before starting the installation process, consider the following points *if you are using a FUSE mount to access
+XRootD:*
 
-If you have not done so, you will need to install the [proper yum repositories](../common/yum):
+-   **User IDs:** If it does not exist already, you will need to create Linux users `xrootd` 
 
+As with all OSG software installations, there are some one-time (per host) steps to prepare in advance:
 
-### Installing XRootD Client
+- Ensure the host has [a supported operating system](/docs/release/supported_platforms)
+- Obtain root access to the host
+- Prepare the [required Yum repositories](/docs/common/yum)
+- Install [CA certificates](/docs/common/ca)
 
-The XRootD client is needed to access XRootD via `xrdcp` or via POSIX preload.
+
+Installing the XRootD Software
+------------------------------
+
+If you are planning on interacting with XRootD using the XRootD client tools  or LD\_PRELOAD, then you'll only need to install the XRootD
+client RPM.
+Mounting a XRootD data store using FUSE will require installing the XRootD FUSE rpm and the client rpm.
+
+### Installing the XRootD Client RPM
+
+The XRootD client rpm is needed to access XRootD using `xrdcp` or POSIX preload.  
+The following steps will install the software on your system.
+
+1. Clean yum cache:
+
+        ::console
+        root@host # yum clean all --enablerepo=\*
+
+1. Update software:
+
+        :::console
+        root@host # yum update
+
+    This command will update **all** packages
+
+1. Install XRootD Client rpm:
+
+        :::console
+        root@host # yum install xrootd-client
```

Comment:
Let's replace `root@host #` with `root@client #` to indicate that this can happen from another host